backend/open_webui/config.py

```diff
@@ -1866,7 +1866,7 @@ class BannerModel(BaseModel):
 WIKIPEDIA_GROUNDING_MAX_CONCURRENT = PersistentConfig(
     "WIKIPEDIA_GROUNDING_MAX_CONCURRENT",
     "rag.wikipedia.grounding.max_concurrent",
-    int(os.getenv("WIKIPEDIA_GROUNDING_MAX_CONCURRENT", "2")),
+    int(os.getenv("WIKIPEDIA_GROUNDING_MAX_CONCURRENT", "1")),
 )
```

Comment:
Do we event want this to be a config, shouldn't it just be a constant/a part of the class?